src/binary_parser.rs

```diff
@@ -19,7 +19,7 @@ pub struct BinaryInfo {
 }
 
 impl BinaryInfo {
-    #[cfg(feature = "unwind")]
+    #[cfg(all(feature = "unwind", target_os = "linux", target_arch = "x86_64"))]
```

Comment:
We also support native unwinding on windows - which I think this might remove. 

(also we should probably aim to get in aarch64 / armv7 unwinding on linux in at some point - but that is out of scope of this PR).